<ui version="4.0" >
 <class>modCalcVlsrDlg</class>
 <widget class="QWidget" name="modCalcVlsrDlg" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>665</width>
    <height>562</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" >
   <property name="margin" >
    <number>9</number>
   </property>
   <property name="spacing" >
    <number>6</number>
   </property>
   <item>
    <widget class="QTabWidget" name="tabWidget" >
     <property name="currentIndex" >
      <number>1</number>
     </property>
     <widget class="QWidget" name="interact" >
      <attribute name="title" >
       <string>Interactive Mode</string>
      </attribute>
      <layout class="QVBoxLayout" >
       <property name="margin" >
        <number>9</number>
       </property>
       <property name="spacing" >
        <number>6</number>
       </property>
       <item>
        <widget class="QGroupBox" name="groupBox" >
         <property name="title" >
          <string>Select Time &amp;&amp; Location</string>
         </property>
         <layout class="QHBoxLayout" >
          <property name="margin" >
           <number>9</number>
          </property>
          <property name="spacing" >
           <number>6</number>
          </property>
          <item>
           <layout class="QGridLayout" >
            <property name="margin" >
             <number>0</number>
            </property>
            <property name="spacing" >
             <number>6</number>
            </property>
            <item row="0" column="0" >
             <widget class="QLabel" name="label" >
              <property name="text" >
               <string>Universal time:</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1" >
             <widget class="QTimeEdit" name="UTBox" />
            </item>
            <item row="1" column="1" >
             <widget class="ExtDateEdit" name="DateBox" />
            </item>
            <item row="1" column="0" >
             <widget class="QLabel" name="label_2" >
              <property name="text" >
               <string>Date:</string>
              </property>
             </widget>
            </item>
            <item row="2" column="0" colspan="2" >
             <spacer>
              <property name="orientation" >
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" >
               <size>
                <width>191</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
          <item>
           <spacer>
            <property name="orientation" >
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" >
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <layout class="QGridLayout" >
            <property name="margin" >
             <number>0</number>
            </property>
            <property name="spacing" >
             <number>6</number>
            </property>
            <item row="1" column="1" >
             <widget class="dmsBox" name="LatitudeBox" />
            </item>
            <item row="2" column="1" >
             <widget class="KLineEdit" name="ElevationBox" />
            </item>
            <item row="1" column="0" >
             <widget class="QLabel" name="label_4" >
              <property name="text" >
               <string>Latitude:</string>
              </property>
             </widget>
            </item>
            <item row="2" column="0" >
             <widget class="QLabel" name="label_5" >
              <property name="text" >
               <string>Elevation (m):</string>
              </property>
             </widget>
            </item>
            <item row="0" column="0" >
             <widget class="QLabel" name="label_3" >
              <property name="text" >
               <string>Longitude:</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1" >
             <widget class="dmsBox" name="LongitudeBox" />
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_2" >
         <property name="title" >
          <string>Select Coordinates of Target</string>
         </property>
         <layout class="QGridLayout" >
          <property name="margin" >
           <number>9</number>
          </property>
          <property name="spacing" >
           <number>6</number>
          </property>
          <item row="2" column="2" >
           <widget class="KLineEdit" name="EpochBox" />
          </item>
          <item row="2" column="0" colspan="2" >
           <widget class="QLabel" name="label_8" >
            <property name="text" >
             <string>Epoch:</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1" colspan="2" >
           <widget class="dmsBox" name="DecBox" />
          </item>
          <item row="0" column="1" colspan="2" >
           <widget class="dmsBox" name="RABox" />
          </item>
          <item row="1" column="0" >
           <widget class="QLabel" name="label_7" >
            <property name="text" >
             <string>Dec:</string>
            </property>
           </widget>
          </item>
          <item row="0" column="0" >
           <widget class="QLabel" name="label_6" >
            <property name="text" >
             <string>RA:</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" >
         <property name="margin" >
          <number>0</number>
         </property>
         <property name="spacing" >
          <number>6</number>
         </property>
         <item>
          <widget class="QGroupBox" name="groupBox_3" >
           <property name="title" >
            <string>Select Input Velocity</string>
           </property>
           <layout class="QVBoxLayout" >
            <property name="margin" >
             <number>9</number>
            </property>
            <property name="spacing" >
             <number>6</number>
            </property>
            <item>
             <widget class="QRadioButton" name="radioVlsr" >
              <property name="text" >
               <string>LSR</string>
              </property>
              <property name="checked" >
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QRadioButton" name="radioVhelio" >
              <property name="text" >
               <string>Heliocentric</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QRadioButton" name="radioVgeo" >
              <property name="text" >
               <string>Geocentric</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QRadioButton" name="radioVtopo" >
              <property name="text" >
               <string>Topocentric</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QGroupBox" name="groupBox_4" >
           <property name="title" >
            <string>Radial Velocities</string>
           </property>
           <layout class="QHBoxLayout" >
            <property name="margin" >
             <number>9</number>
            </property>
            <property name="spacing" >
             <number>6</number>
            </property>
            <item>
             <spacer>
              <property name="orientation" >
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" >
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <layout class="QGridLayout" >
              <property name="margin" >
               <number>0</number>
              </property>
              <property name="spacing" >
               <number>6</number>
              </property>
              <item row="3" column="0" >
               <widget class="QLabel" name="textLabel3" >
                <property name="text" >
                 <string>Topocentric velocity:</string>
                </property>
               </widget>
              </item>
              <item row="1" column="1" >
               <widget class="KLineEdit" name="VHelioBox" />
              </item>
              <item row="3" column="1" >
               <widget class="KLineEdit" name="VTopoBox" />
              </item>
              <item row="2" column="1" >
               <widget class="KLineEdit" name="VGeoBox" />
              </item>
              <item row="0" column="0" >
               <widget class="QLabel" name="azLabel" >
                <property name="text" >
                 <string>VLSR:</string>
                </property>
               </widget>
              </item>
              <item row="2" column="0" >
               <widget class="QLabel" name="textLabel2" >
                <property name="text" >
                 <string>Geocentric velocity:</string>
                </property>
               </widget>
              </item>
              <item row="1" column="0" >
               <widget class="QLabel" name="textLabel1" >
                <property name="text" >
                 <string>Heliocentric velocity:</string>
                </property>
               </widget>
              </item>
              <item row="0" column="1" >
               <widget class="KLineEdit" name="VLSRBox" />
              </item>
             </layout>
            </item>
            <item>
             <spacer>
              <property name="orientation" >
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" >
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" >
         <property name="margin" >
          <number>0</number>
         </property>
         <property name="spacing" >
          <number>6</number>
         </property>
         <item>
          <spacer>
           <property name="orientation" >
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" >
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="KPushButton" name="ComputeButton" >
           <property name="text" >
            <string>Compute</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer>
           <property name="orientation" >
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" >
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="KPushButton" name="ClearButton" >
           <property name="text" >
            <string>Clear</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer>
           <property name="orientation" >
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" >
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <spacer>
         <property name="orientation" >
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" >
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="batch" >
      <attribute name="title" >
       <string>Batch Mode</string>
      </attribute>
      <layout class="QVBoxLayout" >
       <property name="margin" >
        <number>9</number>
       </property>
       <property name="spacing" >
        <number>6</number>
       </property>
       <item>
        <widget class="QGroupBox" name="groupBox_5" >
         <property name="title" >
          <string>Select Parameters in Input File</string>
         </property>
         <layout class="QVBoxLayout" >
          <property name="margin" >
           <number>9</number>
          </property>
          <property name="spacing" >
           <number>6</number>
          </property>
          <item>
           <layout class="QGridLayout" >
            <property name="margin" >
             <number>0</number>
            </property>
            <property name="spacing" >
             <number>6</number>
            </property>
            <item row="3" column="0" >
             <widget class="QCheckBox" name="DecCheckBatch" >
              <property name="text" >
               <string>Dec:</string>
              </property>
              <property name="checked" >
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item row="3" column="3" >
             <widget class="KLineEdit" name="EpochBoxBatch" />
            </item>
            <item row="2" column="2" >
             <widget class="QCheckBox" name="ElevationCheckBatch" >
              <property name="text" >
               <string>Elevation (m):</string>
              </property>
              <property name="checked" >
               <bool>false</bool>
              </property>
             </widget>
            </item>
            <item row="2" column="0" >
             <widget class="QCheckBox" name="RACheckBatch" >
              <property name="text" >
               <string>RA:</string>
              </property>
              <property name="checked" >
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item row="0" column="3" >
             <widget class="dmsBox" name="LongitudeBoxBatch" />
            </item>
            <item row="3" column="2" >
             <widget class="QCheckBox" name="EpochCheckBatch" >
              <property name="text" >
               <string>Epoch:</string>
              </property>
             </widget>
            </item>
            <item row="0" column="2" >
             <widget class="QCheckBox" name="LongCheckBatch" >
              <property name="text" >
               <string>Longitude:</string>
              </property>
              <property name="checked" >
               <bool>false</bool>
              </property>
             </widget>
            </item>
            <item row="2" column="3" >
             <widget class="KLineEdit" name="ElevationBoxBatch" />
            </item>
            <item row="1" column="2" >
             <widget class="QCheckBox" name="LatCheckBatch" >
              <property name="text" >
               <string>Latitude:</string>
              </property>
              <property name="checked" >
               <bool>false</bool>
              </property>
             </widget>
            </item>
            <item row="0" column="1" >
             <widget class="QTimeEdit" name="UTBoxBatch" />
            </item>
            <item row="1" column="1" >
             <widget class="ExtDateEdit" name="DateBoxBatch" />
            </item>
            <item row="2" column="1" >
             <widget class="dmsBox" name="RABoxBatch" >
              <property name="enabled" >
               <bool>false</bool>
              </property>
             </widget>
            </item>
            <item row="3" column="1" >
             <widget class="dmsBox" name="DecBoxBatch" >
              <property name="enabled" >
               <bool>false</bool>
              </property>
             </widget>
            </item>
            <item row="0" column="0" >
             <widget class="QCheckBox" name="UTCheckBatch" >
              <property name="text" >
               <string>UT:</string>
              </property>
              <property name="checked" >
               <bool>false</bool>
              </property>
             </widget>
            </item>
            <item row="1" column="3" >
             <widget class="dmsBox" name="LatitudeBoxBatch" />
            </item>
            <item row="1" column="0" >
             <widget class="QCheckBox" name="DateCheckBatch" >
              <property name="text" >
               <string>Date:</string>
              </property>
              <property name="checked" >
               <bool>false</bool>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" >
            <property name="margin" >
             <number>0</number>
            </property>
            <property name="spacing" >
             <number>6</number>
            </property>
            <item>
             <widget class="QCheckBox" name="InputVelocityCheckBatch" >
              <property name="text" >
               <string>Input velocity:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="InputVelocityComboBatch" >
              <item>
               <property name="text" >
                <string>Heliocentric</string>
               </property>
              </item>
              <item>
               <property name="text" >
                <string>Geocentric</string>
               </property>
              </item>
              <item>
               <property name="text" >
                <string>Topocentric</string>
               </property>
              </item>
              <item>
               <property name="text" >
                <string>LSR</string>
               </property>
              </item>
             </widget>
            </item>
            <item>
             <widget class="KLineEdit" name="InputVelocityBoxBatch" />
            </item>
            <item>
             <spacer>
              <property name="orientation" >
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" >
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_6" >
         <property name="title" >
          <string>Select Parameters for Output File</string>
         </property>
         <layout class="QHBoxLayout" >
          <property name="margin" >
           <number>9</number>
          </property>
          <property name="spacing" >
           <number>6</number>
          </property>
          <item>
           <widget class="QRadioButton" name="AllRadioBatch" >
            <property name="text" >
             <string>All parameters</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="InputRadioBatch" >
            <property name="text" >
             <string>Only parameters in input file</string>
            </property>
            <property name="checked" >
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_7" >
         <property name="title" >
          <string>Select Filenames</string>
         </property>
         <layout class="QGridLayout" >
          <property name="margin" >
           <number>9</number>
          </property>
          <property name="spacing" >
           <number>6</number>
          </property>
          <item row="1" column="1" >
           <widget class="KUrlRequester" name="OutputFileBoxBatch" />
          </item>
          <item row="0" column="1" >
           <widget class="KUrlRequester" name="InputFileBoxBatch" />
          </item>
          <item row="1" column="0" >
           <widget class="QLabel" name="label_10" >
            <property name="text" >
             <string>Output file:</string>
            </property>
           </widget>
          </item>
          <item row="0" column="0" >
           <widget class="QLabel" name="label_9" >
            <property name="text" >
             <string>Input file:</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" >
         <property name="margin" >
          <number>0</number>
         </property>
         <property name="spacing" >
          <number>6</number>
         </property>
         <item>
          <spacer>
           <property name="orientation" >
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" >
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="KPushButton" name="RunButtonBatch" >
           <property name="text" >
            <string>Run</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer>
           <property name="orientation" >
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" >
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <spacer>
         <property name="orientation" >
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" >
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="overview" >
      <attribute name="title" >
       <string>Overview</string>
      </attribute>
      <layout class="QVBoxLayout" >
       <property name="margin" >
        <number>9</number>
       </property>
       <property name="spacing" >
        <number>6</number>
       </property>
       <item>
        <widget class="KTextBrowser" name="OverviewBox" />
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KUrlRequester</class>
   <extends>QFrame</extends>
   <header>kurlrequester.h</header>
  </customwidget>
  <customwidget>
   <class>ExtDateEdit</class>
   <extends>QSpinBox</extends>
   <header>extdatetimeedit.h</header>
  </customwidget>
  <customwidget>
   <class>KLineEdit</class>
   <extends>QLineEdit</extends>
   <header>klineedit.h</header>
  </customwidget>
  <customwidget>
   <class>KPushButton</class>
   <extends>QPushButton</extends>
   <header>kpushbutton.h</header>
  </customwidget>
  <customwidget>
   <class>dmsBox</class>
   <extends>KLineEdit</extends>
   <header>widgets/dmsbox.h</header>
  </customwidget>
  <customwidget>
   <class>KTextBrowser</class>
   <extends>QTextBrowser</extends>
   <header>ktextbrowser.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
